### PR TITLE
Replace links to `buildbot2.r-l.o` with `bors.r-l.o`

### PR DESCRIPTION
**Pull request by JohnTitor**
We've migrated to bors.rust-lang.org, thanks to Pietro :)

### CODE REVIEW

**Review by LeSeulArtichaut — APPROVED** [review]
Sweet!

File: src/contributing.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ with one another are marked as "always roll up".
 [rust-highfive]: https://github.com/rust-highfive
 [steveklabnik]: https://github.com/steveklabnik
 [@bors]: https://github.com/bors
-[merge-queue]: https://buildbot2.rust-lang.org/homu/queue/rust
+[merge-queue]: https://bors.rust-lang.org/homu/queue/rust
```
Comment (camelid) [inline src/contributing.md:131]: This redirects to /queue/rust, so it should be:

```suggestion
[merge-queue]: https://bors.rust-lang.org/queue/rust
```

File: src/getting-started.md [hunk 2]
```diff
@@ -408,7 +408,7 @@ a PR (there are few other commands, but they are less relevant here). This puts
 the PR in [bors's queue][bors] to be tested and merged. Be patient; this can take a
 while and the queue can sometimes be long. PRs are never merged by hand.
 
-[bors]: https://buildbot2.rust-lang.org/homu/queue/rust
+[bors]: https://bors.rust-lang.org/homu/queue/rust
```
Comment (camelid) [inline src/getting-started.md:411]: Same here.

```suggestion
[bors]: https://bors.rust-lang.org/queue/rust
```

File: src/tests/intro.md [hunk 3]
```diff
@@ -132,7 +132,7 @@ only a subset run the full suite of tests (see Rust's [platform tiers]).
 [GitHub Actions]: https://github.com/rust-lang/rust/actions
 [rust-lang-ci]: https://github.com/rust-lang-ci/rust/actions
 [bors]: https://github.com/servo/homu
-[queue]: https://buildbot2.rust-lang.org/homu/queue/rust
+[queue]: https://bors.rust-lang.org/homu/queue/rust
```
Comment (camelid) [inline src/tests/intro.md:135]: And here.

```suggestion
[queue]: https://bors.rust-lang.org/queue/rust
```

File: src/compiler-team.md [hunk 4]
```diff
@@ -82,7 +82,7 @@ offer r+ privileges. This means that you have the right to instruct
 merge a PR
 ([here are some instructions for how to talk to bors][homu-guide]).
 
-[homu-guide]: https://buildbot2.rust-lang.org/homu/
+[homu-guide]: https://bors.rust-lang.org/homu/
```
Comment (camelid) [inline src/compiler-team.md:85]: I missed this one :)

```suggestion
[homu-guide]: https://bors.rust-lang.org/
```

File: src/contributing.md (hunk 1 above)
Comment (JohnTitor) [reply src/contributing.md]: Good point! Fixed them.

**Pull request merged**